<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_AppSettings</class>
 <widget class="QDialog" name="Dialog_AppSettings">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>253</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>510</width>
    <height>253</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>510</width>
    <height>253</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>491</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>511</width>
     <height>181</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_diskcacheStorageRootPath">
    <attribute name="title">
     <string>Disk cache storage</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_cacheStorageRootPath">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>321</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_cacheStorageRootPath_selectDirectory">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>select directory...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_cacheStorageRootPath_comment">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>461</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>This directory stores internal application data about used images. Select a dedicated existing directory, which has read and write permissions for the user.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_imageView">
    <attribute name="title">
     <string>Image view</string>
    </attribute>
    <widget class="QLabel" name="label_ImageView_RAMcomment">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>481</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>This value defines the RAM amount used to view one of the input images. The default value is 512.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_unitMB">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>MegaBytes</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_ImageView_RAM">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RAM for one input image:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="SpinBox_withoutTextEdit" name="widget_spinBoxWithoutTextEdit_imageView_RAMAmount" native="true">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>17</y>
       <width>81</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpinBox_withoutTextEdit</class>
   <extends>QWidget</extends>
   <header>spinbox_withoutTextEdit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_AppSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_AppSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
